process/request_resources.md

```diff
@@ -0,0 +1,60 @@
+
+### Swag
+
+The OpenSSF Marketing Committee keeps a small, annual swag stipend for all projects. Projects may wish to seek outside funding for project swag beyond this stipend.
+
+## Requests through the OpenSSF Budget Committee
+Requests that go through the OpenSSF Budget Committee are reviewed at regularly occuring Budget Committee meetings. Projects should plan their funding needs and requests with this timeline in mind.
+
+### Infrastructure
+
+_Program TBD as of June 2022. Intended program details below._
+Central infrastructure is available to qualifying projects for test and build support. Projects should submit requests on an annual basis to the OpenSSF Budget Committee, a minimum of three months before the requested start date. Projects are responsible for submitting a resource forecast with their request. The Budget Committee and project will work together to agree on minimum infrastructure support based on annual budget. Projects are responsible for communicating to the Budget Committee if there are anticipated increases (including spike events) arise. 
+
+### Maintainer stipends
+
+_Program TBD as of June 2022. Intended program details below._
+Limited, time-bound maintainer stipends for specified project improvements may be available to qualifying projects. Projects should submit a request to the OpenSSF Budget Committee with proposed scope, cost, desired outcome, TAC sponsor approval, and timeline at least three months before the requested start date. Maintainer stipends may not be paid to employees of OpenSSF member companies (excludes Associate members).
+
+### Security audits and improvements
+
+Qualifying projects may request funding for security audits and other services offered by third party security professionals. Before creating a request for proposals, projects should confirm funding availability with the OpenSSF Budget Committee. After receiving proposals, projects should request their selected proposal be funded through the Budget Committee. Projects who need assistance creating an RFP should contact the OpenSSF Planning Committee. RFPs must be posted in a public channel(s) (GitHub issue, mailing list). OpenSSF member companies may bid on RFPs. 
+
+## Other
+
+### Code of Conduct support
+
+All projects must follow the [OpenSSF Code of Conduct](https://openssf.org/community/code-of-conduct/), or, if a project has its own Code of Conduct, it must be reviewed and approved by the TAC. 
+
+If projects receive reports of conduct violations, they should report them to conduct@openssf.org.
```

Comment:
Should or must?

Reply:
I'd suggest removing this sentence and leaving it simply "they must follow the CoC".